panel/1.0/webserver_configuration.md

```diff
@@ -109,6 +113,50 @@ sudo a2enmod rewrite
 sudo systemctl restart apache2
 ```
 
+:::
+::: tab "Caddy With Automatic SSL"
+
+You should paste the contents of the file bellow, replacing `<domain>` with your domain name.
+
+The default config path is `/etc/caddy/Caddyfile`.
```

Comment:
Should be deleted? As default NGINX/Apache do

Reply:
Yes, perhaps, yet I consider reader to be naive, therefore more explanatory docs the better IMO

Reply:
I mean, probably creating a new file would be better?

Reply:
So something like ... ?

```txt
```shell
mkdir -p /etc/caddy
editor /etc/caddy/Caddyfile
```
```

Reply:
Meant, to delete the default configuration (or perhaps Caddy has a disable command, like apache2) and make e.g. `/etc/caddy/Pterodactyl`?

Reply:
Not a good idea. Since Caddy's systemd service has the `--config /etc/caddy/Caddyfile` flag. Changing it brings a lot of complexity to the setup, managing custom systemd service. 

I would not do that.

Reply:
Alright, but by default, is the file empty (or doesn't exist)?

Reply:
TBH I do not remember. I think the file does exist and contains some basic webserver, an example of sorts.

Reply:
Well if it does, it should be erased and then the user can setup the webserver

Reply:
So adding a command or note to delete everything created by default and than writing own config?

Reply:
> So adding a command or note to delete everything created by default and than writing own config?

Once again, if the config does exist and it isn't empty, yea. It should be erased and then the user will once again make it to paste the config